tests/cloner_test.go

```diff
@@ -575,6 +577,102 @@ var _ = Describe("all clone tests", func() {
+
+					dataVolume := utils.NewDataVolumeWithHTTPImportAndStorageSpec(dataVolumeName, "200Mi", fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs))
+					dataVolume.Spec.Storage.VolumeMode = &sourceVolumeMode
+					if sourceSCName != "" {
+						dataVolume.Spec.Storage.StorageClassName = &sourceSCName
+					}
+					dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+					Expect(err).ToNot(HaveOccurred())
+					f.ForceBindPvcIfDvIsWaitForFirstConsumer(dataVolume)
+					sourcePvc, err = f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Wait for source DV Succeeded phase")
+					err = utils.WaitForDataVolumePhaseWithTimeout(f, f.Namespace.Name, cdiv1.Succeeded, dataVolumeName, cloneCompleteTimeout)
+					Expect(err).ToNot(HaveOccurred())
+
+					// We attempt to create the sizeless DV
+					targetDV := utils.NewDataVolumeForCloningWithEmptySize("target-dv", sourcePvc.Namespace, sourcePvc.Name, nil, &targetVolumeMode)
+					if targetSCName != "" {
+						targetDV.Spec.Storage.StorageClassName = &targetSCName
+					}
+					controller.AddAnnotation(targetDV, controller.AnnDeleteAfterCompletion, "false")
+					targetDataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, targetDV)
+					Expect(err).ToNot(HaveOccurred())
+					targetPvc, err = utils.WaitForPVC(f.K8sClient, targetDataVolume.Namespace, targetDataVolume.Name)
+					Expect(err).ToNot(HaveOccurred())
+					f.ForceBindPvcIfDvIsWaitForFirstConsumer(targetDataVolume)
+
+					By("Wait for target DV Succeeded phase")
+					err = utils.WaitForDataVolumePhaseWithTimeout(f, f.Namespace.Name, cdiv1.Succeeded, "target-dv", cloneCompleteTimeout)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Source file system pvc md5summing")
+					sourceMD5, err := f.GetMD5(f.Namespace, sourcePvc, sourceDiskImagePath, crossVolumeModeCloneMD5NumBytes)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Deleting verifier pod")
+					err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
+					Expect(err).ToNot(HaveOccurred())
+					_, err = utils.WaitPodDeleted(f.K8sClient, utils.VerifierPodName, f.Namespace.Name, verifyPodDeletedTimeout)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Target file system pvc md5summing")
+					targetMD5, err := f.GetMD5(f.Namespace, targetPvc, targetDiskImagePath, crossVolumeModeCloneMD5NumBytes)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Deleting verifier pod")
+					err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Checksum comparison")
+					Expect(sourceMD5).To(Equal(targetMD5))
+				},
+				Entry("Block to block (empty storage size)", v1.PersistentVolumeBlock, v1.PersistentVolumeBlock),
+				Entry("Block to filesystem (empty storage size)", v1.PersistentVolumeBlock, v1.PersistentVolumeFilesystem),
```

Comment:
Aren't you skipping this combination on line 606?

Reply:
Yeah, maybe that wasn't too clear. I included it as an `Entry` because that should be a perfectly valid case to test, but since we have an unrelated bug that keeps the case from working, I added a provisional `Skip` until we fix it.

Reply:
Oh okay what bug is that? Is that where the sizes don't line up?

Reply:
This one: https://bugzilla.redhat.com/show_bug.cgi?id=2079781 (I think @maya-r is working on it)

Reply:
Nevermind, I just saw that that bug is for the opposite case, my fault

Reply:
This is the bug I was talking about: https://bugzilla.redhat.com/show_bug.cgi?id=2033191. It says that it happens when using PVC API, but it's also happening to me with Storage. I think @maya-r is working on it.

Reply:
Actually, the description of the bug implies that my case could be different, so I don't know if I'm missing something. In any case, the bug that I'm having can also be reproducible in this previously written [test](https://github.com/kubevirt/containerized-data-importer/blob/main/tests/cloner_test.go#L425), when changing the clone size so it matches the original one:

https://github.com/kubevirt/containerized-data-importer/blob/8db48708fc30a757cb86855bda05bda08eaa8ba0/tests/cloner_test.go#L440

Reply:
I might be flooding here, but after adding `ReadWriteOnce` to the source PVC, the case works. Since I was using Storage API in both source and target I didn't know I needed something like this, so I guess it's my fault.

Reply:
So if you are skipping due to that bug, I would remove the manual skip code above and make this a PEntry (which will skip the entry).